docs/sources/koji.rst

```diff
@@ -77,6 +78,21 @@ Push items produced in this case may include:
 - :class:`~pushsource.OperatorManifestPushItem` - if images have attached operator
   manifest archives
 
+Acessing virtual machine images
+...............................
+
+Use the ``vmi_build`` parameter to request virtual machine images from one or
+more builds. Builds can be specified by NVR. Builds should be produced by
+`OSBS`_ or should use compatible metadata.
```

Comment:
> Builds should be produced by OSBS or should use compatible metadata.

Is this true?  OSBS is producing these builds?

Reply:
OSBS builds just container images
* https://github.com/containerbuildsystem

They have good upstream documentation about metadata
* https://github.com/containerbuildsystem/atomic-reactor/blob/master/docs/koji.md

But AMI, VHD and image for google cloud shoudl be build by image builder

Reply:
I'll adjust it to ImageBuilder, thanks!

Reply:
Fixed, PTAL again

Reply:
TYVM for change. And just for the record (your info)
It is following github organization for the upstream
* https://github.com/osbuild

Reply:
Do you think it is a good idea to rename to `osbuild` or at least point a link to https://www.osbuild.org/ in the docs as well?

Reply:
I asked one of developers and `osbuild` is just a name of one component.
And "image builder" is a suite of projects which allows to build image as a service

And they use "image builder" in case of koji integration in the docs which you shared.
* https://www.osbuild.org/guides/image-builder-service/image-builder-koji.html

Reply:
Thanks!